addon/types/model_export/model.py

```diff
@@ -34,7 +35,7 @@ def __init__(self, game, model):
             directory = self.modelsrc.joinpath(self.name)
         self.directory = common.verify_folder(directory)
 
-        studiomdl = self.bin.joinpath('studiomdl.exe')
+        studiomdl = game_utils.get_studiomdl_path(game)
         quickmdl = self.bin.joinpath('quickmdl.exe')
         self.studiomdl = quickmdl if quickmdl.is_file() else studiomdl
```

Comment:
Let's move the QuickMDL thing into get_studiomdl_path too.

Reply:
Does QuickMDL support Linux too?

Reply:
Oh I don't know actually.. I assume it works through Wine but don't know if there's a native binary.